sdk/identity/identity/src/client/errors.ts

```diff
@@ -119,11 +119,11 @@ export class AuthenticationError extends Error {
     }
 
     super(
-      `An error was returned while authenticating to Azure Active Directory (status code ${statusCode}).\n\nMore details:\n\n${JSON.stringify(
-        errorResponse,
-        null,
-        "  "
-      )}`
+      `${
+        JSON.parse(JSON.stringify(errorResponse, null, "  ")).error
```

Comment:
I don't understand why `errorResponse` is being roundtripped through JSON.stringify and JSON.parse